doc/api.rst

```diff
@@ -18,13 +18,13 @@ Simulation (:py:mod:`hnn_core`):
    L5Pyr
    L2Basket
    L5Basket
-   ExtFeed
    simulate_dipole
    Network
+   NetworkDrive
```

Comment:
@jasmainak you may disagree?

Reply:
umm ... no strong feeling. The `NetworkDrive` object is a bit ill-defined at the moment but I see the value of the documentation being easily accessible from the API page.